cms/templates/studio_xblock_wrapper.html

```diff
@@ -2,13 +2,17 @@
 <%!
 from django.utils.translation import ugettext as _
 from cms.djangoapps.contentstore.views.helpers import xblock_studio_url
-from cms.djangoapps.contentstore.utils import is_visible_to_specific_partition_groups
+from cms.djangoapps.contentstore.utils import is_visible_to_specific_partition_groups, get_editor_page_base_url
 from lms.lib.utils import is_unit
 from openedx.core.djangolib.js_utils import (
     dump_js_escaped_json, js_escaped_string
 )
+from cms.djangoapps.contentstore.toggles import use_new_text_editor, use_new_problem_editor, use_new_video_editor
 %>
 <%
+use_new_editor_text = use_new_text_editor()
```

Comment:
@doctoryes you might notice that the imports, which are compiled statically, are in a <%!%> like so: https://docs.makotemplates.org/en/latest/syntax.html#module-level-blocks whereas this line is run each time an instance of the template is aquired (every time an xblock is fetch to be displayed in a unit page).

Reply:
Thanks, @connorhaugh ! I also tested it and confirmed that the waffle flag is served dynamically.